rastervision_pytorch_learner/rastervision/pytorch_learner/learner.py

```diff
@@ -211,37 +215,75 @@ def setup_model(self, model_def_path: str = None) -> None:
         """
         extCfg = self.cfg.model.external_def
         if extCfg is not None:
-            hubconf_dir_from_cfg = get_hubconf_dir_from_cfg(
-                extCfg, parent=self.modules_dir)
-            if isdir(hubconf_dir_from_cfg) and not extCfg.force_reload:
-                hubconf_dir = hubconf_dir_from_cfg
-            else:
-                hubconf_dir = model_def_path
-
+            hubconf_dir = self._get_external_module_dir(extCfg, model_def_path)
             self.model = self.load_external_module(
                 extCfg=extCfg, hubconf_dir=hubconf_dir)
         else:
             self.model = self.build_model()
-
         self.model.to(self.device)
-
         self.load_init_weights()
 
     @abstractmethod
     def build_model(self) -> nn.Module:
         """Build a PyTorch model."""
         pass
 
+    def setup_loss(self, loss_def_path: Optional[str] = None) -> None:
+        """Setup self.loss.
+
+        Args:
+            loss_def_path (str, optional): Loss definition path. Will be
+            available when loading from a bundle. Defaults to None.
+        """
+        extCfg = self.cfg.solver.external_loss_def
```

Comment:
Is there a reason why `extCfg` isn't named `ext_cfg` (following the usual Python naming conventions)?

Reply:
That's a good point. I have no idea why I didn't notice that before.